xml/schema/layout-5-5-5.xsd

```diff
@@ -55,6 +55,7 @@ signals - obsolete, EMPTY, never used
   <xs:include schemaLocation="http://jmri.org/xml/schema/types/lights-4-19-2.xsd"/>
   <xs:include schemaLocation="http://jmri.org/xml/schema/types/reporters-2-9-6.xsd"/>
   <xs:include schemaLocation="http://jmri.org/xml/schema/types/memories-2-9-6.xsd"/>
+  <xs:include schemaLocation="http://jmri.org/xml/schema/types/stringios-5-11-2.xsd"/>
```

Comment:
The referenced file is not included in the PR.

Reply:
Thanks. I've added it.